main/src/vtr_gui/vtr_gui/vtr-gui/src/components/graph/GraphMap.js

```diff
@@ -226,6 +330,7 @@ class GraphMap extends React.Component {
           center={[map_center.lat, map_center.lng]}
           // center={[43.6605, -79.3964]} /* Jordy Modification For PETAWAWA center={[45.8983, -77.2829]} => TODO We should make this set dynamically from the yaml config*/
           zoom={18}
+          doubleClickZoom={false}
```

Comment:
This change should be recorded in the final commit log.

Reply:
Fixed